addons/tmf_loadouts/CfgLoadouts.hpp

```diff
@@ -53,6 +53,12 @@ class CfgLoadouts {
 		tooltip = "Author: Freddo";
 		#include "loadouts\ru_army_2010.hpp"
 	};
+	class ussr_vdv_1980 {
```

Comment:
And now you accidentally added the VDV loadout to the Aussie branch. Get on TS and I'll show you how to manage branches in GitKraken.